docs/pages/blog/turbo-0-4-0.mdx

```diff
@@ -88,29 +88,29 @@ RUN npm i -g turbo
 
 # Prune the workspace for the `frontend` app
 FROM base as pruner
-COPY .git ./.git
+WORKDIR /app
 COPY . .
-RUN turbo prune --scope=frontend
+RUN turbo prune --scope=frontend --docker
 
 # Add pruned lockfile and package.json's of the pruned subworkspace
-FROM pruner AS installer
-COPY --from=builder /app/out/json/ .
-COPY --from=builder /app/out/yarn.lock ./yarn.lock
+FROM base AS installer
+COPY --from=pruner /app/out/json/ .
+COPY --from=pruner /app/out/yarn.lock ./yarn.lock
 # Install only the deps needed to build the target
 RUN yarn install
 
 # Copy source code of pruned subworkspace and build
-FROM installer as builder
```

Comment:
Since we're already running `COPY --from=installer /app/ .` within this image we will likely also want to base this on `base` (as to not copy all the files from the `installer` image by default)